index.d.ts

```diff
@@ -0,0 +1,17 @@
+import {Node, Parent} from 'unist'
+import {Test} from 'unist-util-is/index'
```

Comment:
Why is the `/index` needed?

Reply:
If I try to import the Test interface from `unist-util-is` without the `/index` dtslint can't find the interface and will error out.

Reply:
@Roang-zero1 could you try removing it again?
Testing locally `dtslint` works with `import {Test} from 'unist-util-is'`

Reply:
Tried again and it worked.